themes/nav-community-v2/layouts/shortcodes/hero_section.html

```diff
@@ -0,0 +1,10 @@
+<div class="hero-section {{ if eq (.Get " gradient ") "true " }} bg-gradient {{ end }}">
+    <img src="{{ .Get "imgSrc" }}" class="hero-img">
+    <div class="text-section-container">
+        <h1 class="">{{ .Get "titleText"}}</h1>
+        <p class="paragraph-text">{{ .Get "paragraphText" }}</p>
+        {{ if (isset  .Params "buttonText") }}
+            <a href="{{ .Get "buttonLink" }}" class="round-btn blue-btn">{{ .Get "buttonText" }}  <img src="{{ .Get "buttonImgSrc" }}" clas="btn-img"> </a></a>
```

Comment:
The button is 100%. We should probably add a new class that limits the width so it isn't so big.
Like make it the size of the text + padding.

Reply:
Wrapped button with a div to fix this